.github/workflows/test.yml

```diff
@@ -27,22 +27,45 @@ jobs:
           skip-go-installation: true
 
   test:
+    name: test
     runs-on: ubuntu-latest
+    
```

Comment:
don't add it in the existing one, create a different job inside the same workflow like `integration-test`